recipes/r-snow/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '0.4-2' %}
```

Comment:
https://anaconda.org/conda-forge/r-snow and many other of this PR already exist.